### PR TITLE
(fix) Custom condition preview not showing up

### LINKED ISSUE #620: Prepare condition: condition preview section is not displayed when prepare a Custom Reporter condition
Related to #350, #616, #610
**Steps:** 
open Prepare Condition page
fill in all the required fields

**AR:** condition preview [section is not displayed](https://drive.google.com/file/d/1SeYhWfY_t0Apo9UlqDfc_7spmGRMh08N/view) (neither duplicating condition nor condition preview)

However, the section appears when prepare an Omen condition, then prepare a Custom reporter condition (see the [video](https://drive.google.com/file/d/14M80Qtp913MzYhsvOJxtR7FboGmEvfzQ/view))

**Expected Result:** section should appear when all the fields are filled in with valid data.

### PR DESCRIPTION
**Pull request by gabitoesmiapodo**
Closes #620

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr624--conditionaltokensexplorer.review.gnosisdev.com

**Review by hpmaxi — APPROVED** [review]

File: src/pages/PrepareCondition/index.tsx [hunk 1]
```diff
@@ -809,8 +809,8 @@ export const PrepareCondition = () => {
           </ErrorContainer>
         )}
         {newCustomConditionStatusInfo &&
-          !customConditionFormHasErrors &&
-          !omenConditionFormHasHerros && (
+          ((conditionType === ConditionType.custom && !customConditionFormHasErrors) ||
+            (conditionType === ConditionType.omen && !omenConditionFormHasHerros)) && (
```
Comment (mariano-aguero) [inline src/pages/PrepareCondition/index.tsx:813]: omenConditionFormHasHerros is this a typo ?

**Review by mariano-aguero — APPROVED** [review]
Please check that possible typo, and then feel free to merge it

File: src/pages/PrepareCondition/index.tsx (hunk 1 above)
Comment (gabitoesmiapodo) [reply src/pages/PrepareCondition/index.tsx:813]: Sure.

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr624--conditionaltokensexplorer.review.gnosisdev.com

**Pull request merged**